src/main/docker/cypress/docker-compose.yml

```
@@ -66,7 +66,7 @@ services:
             - CYPRESS_editorGroupName
             - CYPRESS_instructorGroupName
         # Wait up to 5 minutes until Artemis has booted
-        command: sh -c "cd /app/artemis/src/test/cypress && chmod 777 /root && npm ci && npm exec -- wait-on -i 1000 -t 300000 http://artemis-app:${SERVER_PORT} && npm run cypress:run"
+        command: sh -c "cd /app/artemis/src/test/cypress && chmod 777 /root && npm ci && npm exec -- wait-on -i 1000 -t 300000 http://artemis-app:${SERVER_PORT} && npm run cypress:run -- --config retries=2"
```

Is this `--` necessary? 
```suggestion
        command: sh -c "cd /app/artemis/src/test/cypress && chmod 777 /root && npm ci && npm exec -- wait-on -i 1000 -t 300000 http://artemis-app:${SERVER_PORT} && npm run cypress:run --config retries=2"
```

---

@Strohgelaender Yes, in my tries it showed that the `--` was still needed to pass the argument on to `cypress:run`. I too was under the impression this changed but think that might have been a yarn thing